.github/workflows/official-pr.yml

```diff
@@ -0,0 +1,63 @@
+name: Create official images PR
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+    paths:
+      - "**/Dockerfile"
+      - "**/architectures"
+      - "**/docker-entrypoint.sh"
+      - "generate-stackbrew-library.sh"
+      - "functions.sh"
+
+jobs:
+  pr:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'nodejs' && github.event.pull_request.merged_by != ''
+
+    steps:
+      - name: Checkout the docker-node repo
+        uses: actions/checkout@v2
+        with:
+          path: docker-node
+          fetch-depth: 50
+
+      - name: Checkout the official-images repo
+        uses: actions/checkout@v2
+        with:
+          path: official-images
+          repository: docker-library/official-images
+
+      - name: Generate Stackbrew for diff
+        run: |
+          cd docker-node
+          ./generate-stackbrew-library.sh > ../official-images/library/node
+
+      - name: Create PR in official-images
+        id: create-pr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_API_TOKEN }}
```

Comment:
I'm 90% sure `secrets.GITHUB_TOKEN` will work here.

Reply:
I am not completely sure. The user needs to be able to push commit to some branch on `nodejs-github-bot/official-images` and open pull requests in `docker-library/official-images`. 

https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token

Reply:
Yeah it definitely doesn't have permission for the first one. If it was a branch in this repo it could potentially work.